sqlscan/sqlscan.go

```diff
@@ -133,5 +205,5 @@ func (api *API) ScanRow(dst interface{}, rows *sql.Rows) error {
 	return errors.WithStack(err)
 }
 
-// DefaultAPI is the default instance of API that is wrapped around the dbscan.DefaultAPI instance.
-var DefaultAPI = NewAPI(dbscan.DefaultAPI)
+// DefaultAPI is the default instance of API that is wrapped around the dbscan.API instance.
+var DefaultAPI = NewAPI(dbscan.NewAPI(dbscan.WithLexer(lexer.NewLexer(':', '$')))) //this or no default value at all
```

Comment:
You define `DefaultAPI` as before, you just don't pass any delimiter sign '$' (or the lexer) completely to dbscan

Reply:
then we need to remove the package level SelectNamed, GetNamed, ExecNamed, because if we wouldn't it would lead to unexpected behaviour and in my opinion it does no look very good when there is a partly exposed `DefaultAPI` so it would be better to just get rid of the default api and the package level stuff because it would reduce confusion between the two interfaces (pgxscan and sqlscan).

Reply:
i.e. when the user would call the `sqlscan.SelectNamed` it would probably run into an null pointer reference because the lexer is not defined.

Reply:
I see. Yes, you are right. Since there is no default delimiter for sqlscan we can't offer package-level functions like `SelectNamed()`, etc. And yeah, let's not add them for the pgxscan package either, for consistency reasons.
But I would like to keep package level exported `DefaultAPI` object since some users might already depend on it and it's convenient to have it. We just need to document that one cannot call `*Named()` methods on `DefaultAPI` in the sqlscan package.